app/components/point-group.jsx

```diff
@@ -202,62 +199,76 @@ const PointGroup = props => {
-                                children="Select as Lead"
-                                onDone={() => {
-                                  const newPointObj = {
-                                    ...point,
-                                    groupedPoints: [soloPoint, ...groupedPoints.filter((e, i) => i !== leadIndex)],
-                                  }
-                                  setPointObj(newPointObj)
-                                  onDone({
-                                    valid: true,
-                                    value: { pointObj: newPointObj },
-                                  })
-                                }}
-                                disabled={false}
-                                disableOnClick={false}
+                  const pointChildren = [
+                    <div className={classes.pointBottomButtons}>
+                      <div className={classes.pointWidthButton}>
+                        <ModifierButton
+                          className={classes.pointWidthButton}
+                          title={`Select as Lead: ${point.subject}`}
+                          children="Select as Lead"
+                          onDone={() => {
+                            const newPoint = (
+                              <Point
+                                key={point._id}
+                                point={point}
+                                groupedPoints={[
+                                  <Point point={soloPoint} />,
+                                  ...groupedPoints
+                                    .filter((e, i) => i !== leadIndex)
+                                    .map(p => <Point key={p._id} point={p} />),
+                                ]}
                               />
-                            </div>
-                            <div className={classes.pointWidthButton}>
-                              <TextButton
-                                className={classes.pointWidthButton}
-                                title={`Remove from Group: ${point.subject}`}
-                                children="Remove from Group"
-                                onDone={() => {
-                                  const newPointObj = {
-                                    ...soloPoint,
-                                    groupedPoints: groupedPoints.filter((e, i) => i !== leadIndex),
-                                  }
-                                  setPointObj(newPointObj)
-                                  onDone({
-                                    valid: true,
-                                    value: { pointObj: newPointObj, removedPointObjs: [point] },
-                                  })
-                                }}
+                            )
+                            setPoint(newPoint)
+                            onDone({
+                              valid: true,
+                              value: { point: newPoint },
```

Comment:
here value: point is being sent, but in the original code it was pointObj. This is a change to the interface spec.  Some parent component is (could be I'm not sure yet) rendering point group and expcting pointObj.  Are we catching the places that are rendering PointGroup and passing an onDone that's expecting a poingObj?